src/containers/Tablet/TabletControls/TabletControls.tsx

```diff
@@ -17,18 +17,19 @@ type VisibleDialogType = EVisibleDialogType | null;
 
 interface TabletControlsProps {
     tablet: TTabletStateInfo;
+    fetchData: VoidFunction;
 }
 
-export const TabletControls = ({tablet}: TabletControlsProps) => {
+export const TabletControls = ({tablet, fetchData}: TabletControlsProps) => {
     const {TabletId, HiveId} = tablet;
 
     const [isDialogVisible, setIsDialogVisible] = useState(false);
     const [visibleDialogType, setVisibleDialogType] = useState<VisibleDialogType>(null);
-    const [isTabletActionsDisabled, setIsTabletActionsDisabled] = useState(false);
+    const [isTabletActionLoading, setIsTabletActionLoading] = useState(false);
 
     // Enable controls after data update
     useEffect(() => {
-        setIsTabletActionsDisabled(false);
+        setIsTabletActionLoading(false);
```

Comment:
instead of this `useEffect` lets make a function
```
const onConfirmActionFinish = () => {
  fetchData()
  setIsTabletActionLoading(false)
}
```

Reply:
In this case buttons loading state resets before data is loaded (tablet state updated). So it will be possible to click "stop" twice

Reply:
To handle this you may pass `tabletDataIsLoading` and set `loading = isTabletActionLoading || tabletDataIsLoading`

Reply:
Now we have different loading state for tablet content and for the buttons.

Buttons loading state is needed to prevent unavailable actions. The thing is that actions is connected with tablet `State` and for different states different set of actions is available. When you trigger an action, tablet `State` could change, so to enable buttons we await actual tablet data with the latests `State`.

Additionally, we have tablet data autoupdate, which happens every 30 seconds. If to pass this state to tablet controls, they will be unavailable every 30 seconds as well. It’s not correct, because it isn’t caused by action that could trigger tablet `State` change.

To sum up:
1) Buttons loading state should be connected with tablet `State` actuality
2) We need actual tablet `State`, but cannot use tablet `loading` to await, because it triggers loading when it isn't needed
3) We cannot reset buttons state in `confirmAction`, because this way we don't have actual `State` when reset

Thus we have two different states with different logic.